src/dotnet-scaffolding/dotnet-scaffold/AspNet/CodeModificationConfigs/blazorIdentityChanges.json

```diff
@@ -131,11 +128,8 @@
           "CheckBlock": "<NotAuthorized>"
         },
         {
-          "ReplaceSnippet": [ "<Router AppAssembly=\"typeof(Program).Assembly\">" ],
-          "MultiLineBlock": [
-            "@using $(IdentityNamespace).Shared",
-            "<Router AppAssembly= \"typeof(Program).Assembly\">"
-          ],
```

Comment:
This wasn't always working because the line containing `<Router ...` changed in the .NET 10 templates. Since all we're trying to do is add a `@using` at the top of the file, I changed this block to just prepend the using directive to the beginning of the file.